.github/workflows/build.yml

```diff
@@ -24,12 +30,13 @@ jobs:
       CIBW_BUILD: ${{ matrix.cibw_build }}
       CIBW_ARCHS_LINUX: ${{ matrix.arch }}
       CIBW_ARCHS_MACOS: "x86_64 arm64"
+      CIBW_ENABLE: true
```

Comment:
the value should be `cpython-freethreading` says https://cibuildwheel.pypa.io/en/stable/options/#enable. that should fix the CI failures.